CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+<!--
+Copyright 2017 F5 Networks Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Contributing Guide for k8s-bigip-ctlr
+If you have found this that means you you want to help us out. Thanks in advance for lending a hand! This guide should get you up and running quickly and make it easy for you to contribute.  
+
+## Issues
+Creating issues is good, creating good issues is even better. Filing meaningful bug reports with lots of information in them helps us figure out what to fix when and how it impacts our users. We like bugs because it means people are using our code, and we like fixing them even more.
+
+Please follow these guidelines for filing issues.
+* Describe the problem
+* Include version of k8s, k8s-bigip-ctlr, and BIG-IP
+* Include detailed information about how to recreate the issue
+* Include relevant and sanitized (no identification) configuration, error messages and logs
+
+## Pull Requests
+The hooks that are automatically run as part of each pull request are:
```

Comment:
I would say:

We use [travis-ci](https://travis-ci.org/F5Networks/k8s-bigip-ctlr) to automatically run some hooks on every pull request.  These must pass before a pull request is accepted.  You can see the details in [.travis.yml](https://github.com/F5Networks/k8s-bigip-ctlr/blob/master/.travis.yml).  If your pull request fails in travis, your pull request will be blocked by travis with a link to the failing run.  Generally, we run these hooks: